runner/operator/access/v1_0_0/snps_and_indels/__init__.py

```diff
@@ -0,0 +1,174 @@
+"""""""""""""""""""""""""""""
+" ACCESS-Pipeline SNV workflow operator
+" http://www.github.com/mskcc/access-pipeline/workflows/snps_and_indels.cwl
+"""""""""""""""""""""""""""""
+
+import json
+from jinja2 import Template
+
+from runner.models import Port
+from runner.operator.operator import Operator
+from runner.serializers import APIRunCreateSerializer
+from file_system.repository.file_repository import FileRepository
+
+
+def get_curated_normals(patient_id):
+    """
+    Return curated normal bams as yaml file objects
+
+    :return: (list, list)
+    """
+    normal_bams = FileRepository.filter(
+        metadata={
+            'patientId': patient_id,
+            'tumorOrNormal': 'Normal',
+            'igocomplete': True
+        }
+    )
+    normal_bams = [{'class': 'File', 'location': b.path} for b in normal_bams]
+    normal_ids = [n['metadata']['cmoSampleId'] for n in normal_bams]
+    return normal_bams, normal_ids
+
+
+def construct_sample_inputs(patient_id, tumor_bam, tumor_simplex_bam, tumor_sample_id, matched_normal_bam, normal_sample_id):
+    with open('runner/operator/access/v1_0_0/snps_and_indels/input_template.json.jinja2') as file:
```

Comment:
It might be better to use something like this to pull the code path: https://github.com/mskcc/beagle/blob/master/runner/operator/argos_operator/v1_1_0/construct_argos_pair.py#L15